terraform/dynamodb.tf

```diff
@@ -1,17 +1,17 @@
 resource "aws_dynamodb_table" "tiled_results_table" {
   name           = substr("${local.project}-tiled_results${local.name_suffix}", 0, 64)
   billing_mode   = "PAY_PER_REQUEST"
-  hash_key       = "analysis_id"
-  range_key      = "tile_id"
+  hash_key       = "tile_id"
+  range_key      = "result_id"
 
   attribute {
-    name = "analysis_id"
+    name = "tile_id"
     type = "S"
   }
 
   attribute {
-    name = "tile_id"
-    type = "S"
+    name = "result_id"
```

Comment:
very minor, but I think a name like `part_number` or `part_id` might be more descriptive.